test/socket.stream.js

```diff
@@ -49,16 +51,21 @@ describe('socket.stream', function(){
               httpMsg.socket._readableState.reading.should.be.true
             }
           }
-          
+
           //conventional node streams emit data events to process zmq stream response
           httpMsg.on('data',function (msg){
             msg.should.be.an.instanceof(Buffer);
-            String(msg).should.equal('<!DOCTYPE html><head><meta charset="UTF-8"></head>'
-              +'<body>'
-                +'<p>derpin over protocols</p>'
-              +'</body>'
-            +'</html>');
-            done();
+            var res = String(msg).split('\n');
+            if (res.length > 1) {
+
+            } else {
+              String(res).should.equal('<!DOCTYPE html><head><meta charset="UTF-8"></head>'
```

Comment:
Turning the array back into a string? Wouldn't that introduce commas in between each argument?

```js
['foo', 'bar'] + '';
```
outputs: `"foo,bar"`